ReSharper.FSharp/TypeProviders.targets

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Are these targets specific to the host projects only? The generic `TypeProviders.targets` name sounds like it might should've beeen imported into the Protocol project as well.

Reply:
> Are these targets specific to the host projects only?

Yes.
We can rename it to "TypeProviders.Host.targets"